File: Python/GradingStudents.py
def gradingStudents(grades):
    # Write your code here
    res = []
    for grade in grades:
        if grade > 37:
            remain = 5 - (grade % 5)
            divide = math.floor(grade/5)
            rounded = 5 * (divide + 1)
            if remain < 3:
                res.append(rounded)
            else:
                res.append(grade)
        else:
            res.append(grade)
    return res

File: C#/GradingStudents.cs
public static List<int> gradingStudents(List<int> grades)
{
    List<int> res = new List<int>();
    
    foreach(int grade in grades){
        
        int remain = 5 - (grade % 5);
        int divide = grade/5;
        int rounded = 5 * (divide + 1);
        
        if(grade > 37){
            
            if(remain < 3){
                
                res.Add(rounded);
            }else{
                
                res.Add(grade);
            }
        }else{
            res.Add(grade);
        }
    }
    return res;
}